config/browser.js

```diff
@@ -12,32 +12,59 @@
 module.exports = {
```

Comment:
it could be written as json, however, it then requires an additional action to parse the data before we can loop over the data.